perseo-main/src/main/java/com/telefonica/iot/perseo/RulesServlet.java

```diff
@@ -108,16 +113,22 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
         Utils.putCorrelatorAndTrans(request);
         response.setCharacterEncoding("UTF-8");
         response.setContentType("application/json;charset=UTF-8");
-        PrintWriter out = response.getWriter();
-        String body = Utils.getBodyAsString(request);
 
-        // Save temporary rules, not triggered by events external to the core
-        TimeRulesStore.getInstance().saveTimeRules(body);
+        try {
+			
+			String body = Utils.getBodyAsString(request);
+			
+			// Save temporary rules, not triggered by events external to the core
+			TimeRulesStore.getInstance().saveTimeRules(body);
+			
+			Result r = RulesManager.make(epService, body);
+			response.setStatus(r.getStatusCode());
+			response.getOutputStream().print(Encode.forHtmlContent(r.getMessage()));			
+        }catch(Exception je) {        				
+			logger.error(String.format("error: %s" ,je));
+			response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
+        }
```

Comment:
Wrong indent

Reply:
Fixed in 05982ba4eefed043b201d8b95eb1c9b6190d2673